src/SarifWeb/Views/Shared/_Layout.cshtml

```diff
@@ -3,15 +3,16 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="The Static Analysis Results Interchange Format (SARIF) Website">
     <title>@ViewBag.Title</title>
-    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet" />
```

Comment:
We might be able to remove this line altogether. Reasoning:
* I do not see "Roboto" referenced in the repo.
* Glancing at the rendered site, I don't see it in use.
* Even if it used "Roboto" is used, it is not required from a brand/design standpoint. It is basically just a normal sans-sarif font.

Reply:
Correction, I see it used in the title. However, based on the 3rd bullet I still don't think it's necessary. I designed the original Figma for this site, and Roboto just happened to be the typeface I choose to represent something sans-serif.

Reply:
yes, sounds good, it was used and since you are the original dev for this part we know is safe to remove it, I will remove this line, and remove the 3 manual usages of 'Roboto' in the file Home.css and Frame.css.